docs/source/tutorial_mor_with_anns.md

```diff
@@ -539,6 +539,127 @@ using a reduced basis, one can apply the
+
+```{code-cell}
+training_set = parameter_space.sample_uniformly(15)
+validation_set = parameter_space.sample_randomly(3)
+test_set = parameter_space.sample_randomly(10)
+```
+
+To check how the two reductors perform, we write a simple function that measures the
+errors and the speedups on a test parameter set:
+
+```{code-cell}
+def compute_errors(rom, reductor):
+    speedups = []
+
+    U = fom.solution_space.empty(reserve=len(test_set))
+    U_red = fom.solution_space.empty(reserve=len(test_set))
+
+    for mu in test_set:
+        tic = time.time()
+        u_fom = fom.solve(mu)[1:]
+        U.append(u_fom)
+        time_fom = time.time() - tic
+
+        tic = time.time()
+        u_red = reductor.reconstruct(rom.solve(mu))[1:]
+        U_red.append(u_red)
+        time_red = time.time() - tic
+
+        speedups.append(time_fom / time_red)
+
+    relative_errors = (U - U_red).norm2() / U.norm2()
+
+    return relative_errors, speedups
+```
+
+We now run the
+{class}`~pymor.reductors.neural_network.NeuralNetworkInstationaryReductor`
+and the
+{class}`~pymor.reductors.neural_network.NeuralNetworkLSTMInstationaryReductor`
+with different parameters and evaluate their performance:
+
+```{code-cell}
+from pymor.reductors.neural_network import NeuralNetworkInstationaryReductor, NeuralNetworkLSTMInstationaryReductor
+
+basis_size = 20
+
+reductor = NeuralNetworkInstationaryReductor(fom, training_set, validation_set, basis_size=basis_size,
+                                             pod_params={'product': product}, ann_mse=None, scale_inputs=True,
+                                             scale_outputs=True)
+rom = reductor.reduce(restarts=0)
+rel_errors, speedups = compute_errors(rom, reductor)
+reductor_lstm = NeuralNetworkLSTMInstationaryReductor(fom, training_set, validation_set, basis_size=basis_size,
+                                                      pod_params={'product': product}, ann_mse=None, scale_inputs=True,
+                                                      scale_outputs=True)
+rom_lstm = reductor_lstm.reduce(restarts=0, number_layers=1, hidden_dimension=25, learning_rate=0.01)
+rel_errors_lstm, speedups_lstm = compute_errors(rom_lstm, reductor_lstm)
+```
+
+We finally print the results:
+
+```{code-cell}
+print('Results for the state approximation:')
+print('====================================')
+print()
+print('Approach by Hesthaven and Ubbiali using feedforward ANNs:')
+print('---------------------------------------------------------')
+print(f'Average relative error: {np.average(rel_errors)}')
+print(f'Median of speedup: {np.median(speedups)}')
+print()
+print('Approach using long short-term memory ANNs:')
+print('-------------------------------------------')
+print(f'Average relative error: {np.average(rel_errors_lstm)}')
+print(f'Median of speedup: {np.median(speedups_lstm)}')
+```
+
+In this example, we observe that the LSTMs perform much better than the feedforward ANNs in terms
+of accuracy and runtime.
```

Comment:
On https://docs.pymor.org/new-instationary-nn-reductor-in-tutorial/tutorial_mor_with_anns.html the speedup of the LSTM ROM is actually smaller ...

Reply:
My local tests showed a different behavior. Interesting. I will adjust the discussion accordingly that the speedup of both methods is in the same order of magnitude but that LSTMs are much more accurate in this example.